Palace/Book/UI/TPPBookCellDelegate.m

```diff
@@ -407,7 +408,9 @@ - (void) requestSyncWithCompletion:(void (^)(BOOL))completion {
 ///   - remoteLocation: remote player location
 ///   - operation: operation block on the selected location
 - (void) chooseLocalLocation:(ChapterLocation *)localLocation orRemoteLocation:(ChapterLocation *)remoteLocation forOperation:(void (^)(ChapterLocation *))operation {
-  if (remoteLocation && (![remoteLocation.description isEqualToString:localLocation.description])) {
+  
+  bool remoteLocationIsNewer = [NSString isDate:remoteLocation.lastSavedTimeStamp moreRecentThan:localLocation.lastSavedTimeStamp with:15];
```

Comment:
Could you add a comment or move 15 to a variable here? Is it `kServerUpdateDelay`?

Reply:
Good point, I could just reuse that variable here